images/pj-rehearse/Dockerfile

```diff
@@ -3,4 +3,5 @@ LABEL maintainer="nmoraiti@redhat.com"
 
 RUN yum install -y git && yum clean all
 ADD pj-rehearse /usr/bin/pj-rehearse
+ADD /usr/bin/oc /usr/bin/oc
```

Comment:
You need to make this source file available in the build.
Example:
https://github.com/openshift/release/blob/8c3c1dd5615f3c341781c3acd0dfe146748fa4eb/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L222-L223